### DISCUSSION TITLE
5.4.0

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#954] [🚀 Releases]
## What's Changed
* Support parameterized field getters in generated interfaces #859 #913 @kobylynskyi
* Add `serialVersionUID` to generated classes that implement `Serializable` #810 #811 @rolevinks
* Don't fail with enums in directive parameters #673 #674 @meistermeier
* Support unknown fields during serialization or deserialization of `GraphQLRequest` #866 @aldib
* Support `modelNameSuffix` suffix in `GraphqlJacksonTypeIdResolver` #914 #915 @YarLyashenko
* [Kotlin] Ability to generate sealed interfaces #868 #869 @kbrooks
* [Kotlin] Add option to add defaults to nullable fields #863 @zmack
* [Kotlin] Add `@JvmStatic` annotation in generated builders #767 #771 @kbrooks
* [Kotlin] Fix `default` value initialization #880 #882 @jxnu-liguobin
* [Scala] Fix generateModelOpenClasses #741 @jxnu-liguobin

**Comment by kobylynskyi** [comment]
Maven and Gradle plugins were released.
@jxnu-liguobin could you please release sbt plugin for 5.4.0 release as well? Thanks!

**Reply by jxnu-liguobin** [reply]
It is expected to be released later this week.
